src/llmcompressor/modifiers/quantization/gptq/base.py

```diff
@@ -68,8 +68,8 @@ class GPTQModifier(Modifier):
 
     :param sequential_update: Whether or not to update weights sequentially by layer,
         True saves on GPU memory, default is True
-    :param targets: list of layer names to compress during GPTQ, or '__ALL__'
```

Comment:
Why remove targets from the docstring? We are still accepting it to build the QuantizationModifier